packages/amplify-graphql-api-construct/src/internal/construct-exports.ts

```diff
@@ -73,7 +73,10 @@ export const getGeneratedResources = (scope: Construct): AmplifyGraphqlApiResour
       tables[resourceName] = currentScope;
       return;
     }
-    if (currentScope instanceof CfnTable) {
+    if (
+      currentScope instanceof CfnTable ||
+      (currentScope instanceof CfnResource && currentScope.cfnResourceType === 'Custom::AmplifyDynamoDBTable')
```

Comment:
When the table is `CfnTable`, the CDK override code is
```TypeScript
    const todoTable = api.resources.cfnResources.cfnTables['Todo'] as CfnTable;
    todoTable.billingMode= 'PROVISIONED';
    todoTable.provisionedThroughput= {
      readCapacityUnits: 5,
      writeCapacityUnits: 5,
    };
```
When the table is `CfnResource`, the CDK override code is
```TypeScript
    const todoTable = api.resources.cfnResources.cfnTables['Todo'] as CfnResource;
    todoTable.addPropertyOverride('billingMode', 'PROVISIONED');
    todoTable.addPropertyOverride('provisionedThroughput', {
      readCapacityUnits: 5,
      writeCapacityUnits: 5,
    });
```